Extensions/StringExtensions.cs

```diff
@@ -0,0 +1,21 @@
+namespace Trxlog2Html.Extensions;
+public static class StringExtensions
+{
+    public static bool IsImage(this string value)
+    {
+        return value.EndsWith(".png");
```

Comment:
The IsImage method currently only checks for the .png extension. Should we also consider .jpg, .jpeg, .gif, and other image formats as valid image types? Additionally, it might be better to use StringComparison.OrdinalIgnoreCase with EndsWith to ignore case sensitivity.